code/modules/reqs/supplypacks.dm

```diff
@@ -582,6 +582,12 @@ WEAPONS
 	contains = list(/obj/item/ammo_magazine/flamer_tank/backtank/X)
 	cost = 600
 
+/datum/supply_packs/weapons/fueltank
+	name = "fuel tank"
```

Comment:
Granted this is in the weapons tab and not the engineering tab, but it might be worthwhile to distinguish the xfuel tank from a normal fuel tank.